packages/riipen-ui/src/components/Tooltip.jsx

```diff
@@ -197,6 +197,7 @@ class Tooltip extends React.Component {
         border-width: ${theme.spacing(2)}px;
         content: "";
         position: absolute;
+        white-space: normal;
```

Comment:
Set whitespace here so that the tooltip arrows are not moved around unexpectedly